sql/parser/rename.go

```diff
@@ -34,5 +34,5 @@ type RenameTable struct {
 }
 
 func (node *RenameTable) String() string {
-	return fmt.Sprintf("RENAME TABLE %s %s", node.Name, node.NewName)
+	return fmt.Sprintf("RENAME TABLE %s %s", encIdent(node.Name), encIdent(node.NewName))
```

Comment:
Rather than all of these `encIdent` calls, perhaps we should introduce a new type:

```
type Ident string

func (i Ident) String() string {
  return encIdent(string(i))
}
```